tests/testthat/testdata/download_testdata.R

```diff
@@ -0,0 +1,85 @@
+# Donwnload OSM datasets and save them for testing
+
+# City name (in English) and river name
+city <- "Bucharest"
+river <- "Dâmbovița"
+# Projected CRS - UTM zone 35N
+epsg_code <- 32635
+# OSM values for the "highway" key
+highways_value <- c("motorway", "primary", "secondary", "tertiary")
+
+.data <- NULL
+
+get_highways <- function(bbox) {
+  highways <- CRiSp::osmdata_as_sf("highway", highways_value, bbox)
+  highways <- highways$osm_polygons |>
+    sf::st_cast("LINESTRING") |>
+    dplyr::bind_rows(highways$osm_lines) |>
+    dplyr::select("highway") |>
+    sf::st_transform(epsg_code)
+
+  highways
+}
+
+get_railways <- function(bbox) {
+  railways <- CRiSp::osmdata_as_sf("railway", "rail", bbox)
+  highways <- railways$osm_lines |>
+    sf::st_geometry() |>
+    sf::st_transform(epsg_code)
+
+  highways
```

Comment:
The function will return the value of the last evaluated expression, even if it was assigned to an object, so this is not necessary to explicitly call `highways`. This applies to the other functions as well.

At least this is what I've been doing so far, I hope I am not mistaken.

Reply:
See this basic example for the same behaviour:

```r
my_function <- function(x) {
  y <- x + 1  # y is created with assignment
}

result <- my_function(5)
result  # Output will be 6
```

Reply:
Thanks for the clarification.  You are definitely right! Using your example, I was fooled by the fact that just calling `my_function(5)` does not print any output (but it is just the printing that is missing, assigning the function output to a variable shows that the value is properly returned).